lasy/utils/openpmd_input.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+import scipy.constants as ct
+
+from .grid import Grid
+
+
+def reorder_array(array, md, dim):
```

Comment:
These 3 functions do 2 things:
- swap axes so the last axis is time.
- convert (in a dummy way) from z representation to t representation.

The second operation (conversion) is not really about reordering. However, in a future PR I will probably move the conversion to a different place (put z-t converters in utils, and give them the option to do dummy). Therefore, I am fine with the changes you propose, this is just FYI.